jans-lock/lock-server/service/src/main/proto/grpc/audit.proto

```
@@ -0,0 +1,93 @@
+syntax = "proto3";
+
+package io.jans.lock.audit;
```

Buf (1.63.0)

Files with package "io.jans.lock.audit" must be within a directory "io/jans/lock/audit" relative to root but were in directory "jans-lock/lock-server/service/src/main/proto/grpc".

(PACKAGE_DIRECTORY_MATCH)